vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/exclusion_rule_util.h

```diff
@@ -14,10 +14,6 @@
 
 namespace ads {
 
-class AdType;
-
-bool DoesAdTypeSupportFrequencyCapping(const AdType& type);
```

Comment:
Filtering by ad type is no longer required as with this change we only pass through the ad types to calculate the eligible ads